habitat_sim/utils/datasets_download.py

```diff
@@ -0,0 +1,249 @@
+        #   "version": data version tag
+        # }
+        "habitat-test-scenes": {
+            "source": "http://dl.fbaipublicfiles.com/habitat/habitat-test-scenes.zip",
+            "package_name": "habitat-test-scenes.zip",
+            "unpack_to": data_path + "../",
+            "root": data_path + "scene_datasets/habitat-test-scenes/",
+            "version": "1.0",
+        },
+        "habitat-example-objects": {
+            "source": "http://dl.fbaipublicfiles.com/habitat/objects_v0.2.zip",
+            "package_name": "objects_v0.2.zip",
+            "unpack_to": data_path + "objects/example_objects/",
+            "root": data_path + "objects/example_objects/",
+            "version": "0.2",
+        },
+        "locobot-merged": {
+            "source": "http://dl.fbaipublicfiles.com/habitat/locobot_merged_v0.2.zip",
+            "package_name": "locobot_merged_v0.2.zip",
+            "unpack_to": data_path + "objects/locobot_merged/",
+            "root": data_path + "objects/locobot_merged/",
+            "version": "0.2",
+        },
+        "mp3d-test-scene": {
+            "source": "http://dl.fbaipublicfiles.com/habitat/mp3d_example.zip",
+            "package_name": "mp3d_example.zip",
+            "unpack_to": data_path + "scene_datasets/mp3d_test/",
+            "root": data_path + "scene_datasets/mp3d_test/",
+            "version": "1.0",
+        },
+        "coda-scene": {
+            "source": "'https://docs.google.com/uc?export=download&id=1Pc-J6pZzXEd8RSeLM94t3iwO8q_RQ853'",
+            "package_name": "coda.zip",
+            "download_pre_args": "--no-check-certificate ",
+            "download_post_args": " -O " + data_path + "coda.zip",
+            "unpack_to": data_path + "scene_datasets/",
+            "root": data_path + "scene_datasets/coda/",
+            "version": "1.0",
+        },
+    }
+
+    # data sources can be grouped for batch commands with a new uid
+    data_groups = {
+        "ci-test-assets": [
+            "habitat-test-scenes",
+            "habitat-example-objects",
+            "locobot-merged",
+            "mp3d-test-scene",
+            "coda-scene",
+        ]
+    }
+
+
+def prompt_yes_no(message):
+    r"""Prints a message and prompts the user for "y" or "n" returning True or False."""
+    print("\n-------------------------")
+    print(message)
+    while True:
+        answer = input("(y|n): ")
+        if answer.lower() == "y":
+            return True
+        elif answer.lower() == "n":
+            return False
+        else:
+            print("Invalid answer...")
+
+
+def clean_data(uid):
+    r"""Deletes the "root" directory for the named data-source."""
+    if not data_sources.get(uid):
+        print(f"Data clean failed, no datasource named {uid}")
+        return
+    root_path = data_sources[uid]["root"]
+    print(f"Cleaning datasource ({uid}) directory: {root_path}")
+    try:
+        shutil.rmtree(root_path)
+    except OSError as e:
```

Comment:
Instead of printing error, you could use the traceback module to generate a nicer error message and traceback. Also weird to output errors to stdout, no?